plugins/vendor/zoom/services/ZoomVendorService.php

```diff
@@ -176,7 +176,7 @@ public function fetchRegistrationPageAction($tokensData, $iv)
 	 */
 	public function localRegistrationPageAction($zoomAccountId)
 	{
-		$isOAuthVerification = self::getInstanceContext();
+		$isOAuthVerification = self::shouldUseOAuthAuthenticationMethod();
```

Comment:
Rename , Zoom calls it oauth
shouldUseOAuthAuthMethod

Reply:
shouldUseOAuthAuthMethod
shouldUseOAuthAuthenticationMethod

its practically the same